src/renderers/testing/__tests__/ReactTestRenderer-test.js

```diff
@@ -306,6 +306,41 @@ describe('ReactTestRenderer', () => {
     ]);
   });
 
+  it('supports unmounting when using refs', () => {
+    class Foo extends React.Component {
+      render() {
+        return <div ref="foo"/>;
```

Comment:
Can we add a linting rule for this? hard to remember if `lint` doesn't complain.